client/src/components/Filters/PriceFilter.js

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+import ToggleButton from "@mui/material/ToggleButton";
+import ToggleButtonGroup from "@mui/material/ToggleButtonGroup";
+
+const DEFAULT_FILTER = {
+  $: false,
+  $$: false,
+  $$$: false,
+  $$$$: false,
+};
+
+function PriceCategoryFilter(restaurant) {
+  return selectedPrices.includes(restaurant.price_category);
+}
+
+export default function PriceFilterToggler() {
+  const [priceFilter, setPriceFilter] = React.useState(DEFAULT_FILTER);
+  const [filters, setFilters] = React.useState(() => []);
+
+  const handleFormat = (event, newFormats) => {
+    setFilters(newFormats);
+  };
+  const selectedPrices = Object.keys(priceFilter).filter(
+    (priceKey) => priceFilter[priceKey]
+  );
```

Comment:
What is this for?

Reply:
I will get rid of it, if it is for a future functionality